### ISSUE TITLE
NVDA not announce if an e-mail is answered/forwarded in Outlook 2013

### ISSUE THREAD

**Issue by andersbp** [nvaccess/nvda#6774]
When I read e-mails, and use the arrow keys to navigate up and Down in my folders, e.g. my Inbox, Sent Post etc., NVDA 2016.04 not read if an e-mail is answered/forwarded.

When I come to an e-mail there is answered/forwarded, NVDA tell that the e-mail contains an attached file.

Operating system: Windows 10 64-bit.

Hope you can fix the issue.

Thanks in advance.

/Anders

**Comment by feerrenrut** [comment]
Closing this as a duplicate. This is an older issue but #6911 contains more information.
